File: Programmers_python/전화번호_목록.py
def solution(phone_book):
    phone_book.sort()

    for i in range(len(phone_book) - 1):
        if len(phone_book[i + 1]) >= len(phone_book[i]):
            if phone_book[i + 1].find(phone_book[i]) == 0:
                return False

    return True

File: Programmers_js/전화번호_목록.js
function solution(phone_book) {
    var answer = true;
    phone_book.sort();
    
    for(let i=1; i<phone_book.length; i++) {
        if(phone_book[i].startsWith(phone_book[i-1])){
            answer = false;
            break;
        }
    }
    
    return answer;
}
